### ISSUE TITLE
bug: edges of window glitch

### ISSUE THREAD

**Issue by tjlawal** [slynch8/10x#2968]
**Version: 1.0.234**

This bug only occurs when I have had 10x running for a couple of hours. I usually don't close all the work spaces I have opened, and recently I noticed the edges of the window have a glitch after 8-10 hours of the window program being open.

It doesn't seem to affect any of the program functionality but its an eye sore when the program is not full screen.

![image](https://github.com/user-attachments/assets/aa87067d-13b0-4e7d-86d3-c890cf448447)

**Comment by slynch8** [comment]
It's not happening for me.

Is this Win10 or Win11, and are you using a remote desktop app? And does it go away when you re-focus or resize the app?

**Comment by tjlawal** [comment]
It's Windows 10. Not using a remote desktop app, it's running on an actual desktop. 

The video below shows the bug in action. https://youtu.be/9iNwdBhcSz8

**Comment by slynch8** [comment]
Thanks for the video. It reminds me of this bug https://github.com/slynch8/10x/issues/907#issuecomment-1264657026
I think it was caused by DirectX using the integrated GPU. Maybe you can try focusing it to use a dedicated GPU if it has one.

**Comment by slynch8** [comment]
Can you give this build a try:
https://www.10xeditor.com/upload/10xBroderFix.zip
(unzip to a folder and run the exe from there)

I've made a change that might fix it, and added some extra logging.

**Comment by tjlawal** [comment]
It’s running on a Ryzen 5 5600x so no integrated gpu. The dedicated gpu running the system is a rtx 2060. Doubt it would be the problem, also just did a driver update so that’s ruled out. 

The build you provided is not available for download, did you take it down? 

Cheers

**Comment by slynch8** [comment]
Sorry about that, I've fixed the link. It should work now.

**Comment by slynch8** [comment]
If you get it again can you check your log file for lines starting with `WindowsMain::`

**Comment by tjlawal** [comment]
Will test and get back to you.

**Comment by tjlawal** [comment]
Hiya, some update for yah

After I downloaded the fix you provided, I restarted the system (its been running a hot minute) and have been using it for 2 weeks and the bug hasn't rear its end since. Now running on the latest official version and everything seems to be working as it should. 

Couldn't tell you what the bug could be but it seems to fix have fixed itself and I never even ran the border fix build you provided.

Cheers.
